src/App.tsx

```diff
@@ -1,14 +1,31 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
-
+import { useDispatch } from 'react-redux';
+import { useAppSelector } from './app/hooks';
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { setTodos } from './features/todos';
+
 export const App: React.FC = () => {
+  const [list, setList] = useState<Todo[]>([]);
+  const dispatch = useDispatch();
```

Comment:
don't introduce the state variable if you have redux state. just remove it because you already select todos from redux state in `TodoList` component